### ISSUE TITLE
About tiny imagenet training problem

### ISSUE THREAD

**Issue by zorrocai** [bearpaw/pytorch-classification#19]
I have modified the imagenet.py by adding the lines after the "create model" codes:
    num_ftrs = model.fc.in_features
    model.fc = nn.Linear(num_ftrs,200)
changing the net to train on tiny imagenet.
But it has a problem: training top1 accuracy is about 79.6110 and top5 accuracy is about 91.6950;
while validation accuracys are only 0.7(top1) and 1.47(top5). Do you have any ideas on this weird problem?

**Comment by taomo** [comment]
https://github.com/tjmoon0104/pytorch-tiny-imagenet/blob/master/val_format.py
